yesod-test/Yesod/Test.hs

```diff
@@ -830,6 +831,16 @@ setUrl url' = do
         , rbdGets = rbdGets rbd ++ H.parseQuery (TE.encodeUtf8 urlQuery)
         }
 
+clickOn :: Yesod site => Query -> YesodExample site ()
```

Comment:
Whichever Github user `@since` is, I bet they get a lot of spurious notifications 🤔